debian/snapd.postinst

```diff
@@ -11,4 +11,10 @@ case "$1" in
         if dpkg --compare-versions "$2" lt-nl "2.0.7"; then
             ldconfig
         fi
+        # start required services
+        if [ -d /run/systemd/system ]; then
+            systemctl daemon-reload
+            systemctl -f enable snapd.refresh.timer snapd.refresh.service snapd.autoimport.service snapd.socket snapd.service snap.mount.service
```

Comment:
Sure, we are running into an issue with the trusty version of systemd being significantly different to the xenial one, though. This is assuming a stable interface in terms of cli, not in terms of side-effects like install location of files.